clientlibs/js/src/UpgradeClient.ts

```diff
@@ -120,7 +120,7 @@ export default class UpgradeClient {
     async markExperimentPoint(
       experimentPoint: string,
       condition = null,
-      status: MARKED_DECISION_POINT_STATUS,
+      status: MARKED_DECISION_POINT_STATUS = MARKED_DECISION_POINT_STATUS.MARKED,
```

Comment:
I think the client library should require its users to supply a value for this parameter.